tests/testthat/test-cre.R

```diff
@@ -1,7 +1,7 @@
 test_that("cre Runs Correctly", {
   # Generate sample data
   set.seed(2021)
-  dataset_cont <- generate_cre_dataset(n = 300, rho = 0, n_rules = 2, p = 10,
+  dataset_cont <- generate_cre_dataset(n = 1000, rho = 0, n_rules = 2, p = 10,
```

Comment:
Please keep the sample size as low as possible. It takes time to test the functions. If you are interested in testing with larger data, please add them under `functional_tests` folder.

Reply:
Using a lower sample size it happens frequently that no causal rule is detected and the wrong output is returned. I have already opened an issue for this problem and it will be solved in my following PR. My plan was to set again these values to lower numbers in the following PR.

Reply:
@riccardocadei, please address that problem, and add new tests. Please do not modify old tests. Thanks.

Reply:
I see your point, I reset the original values (but I also add the same tests with a larger dataset size such that at least a causal rule can be detected). Then, should I fix the 'No Causal Rules detected' issue in this PR?

Reply:
Without fixing that Issue here, the tests with sample_size = 200/300 fail.